guides/micronaut-graphql-todo/micronaut-graphql-todo.adoc

```diff
@@ -0,0 +1,398 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+In this guide, we will create a Micronaut application written in @language@ that uses https://graphql.org/[GraphQL] to create a todo application.
+
+____
+GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data. GraphQL provides
+a complete and understandable description of the data in your API, gives clients the power to ask for exactly what they
+need and nothing more, makes it easier to evolve APIs over time, and enables powerful developer tools.
+____
+
+We will be using:
+
+* A Postgres instance running in Docker.
+* https://micronaut-projects.github.io/micronaut-data/latest/guide/[Micronaut-data] to persist our ToDos to this database.
+* https://micronaut-projects.github.io/micronaut-flyway/latest/guide/[Flyway] to handle our database migrations.
+* https://micronaut-projects.github.io/micronaut-graphql/latest/guide/[Micronaut GraphQL] to expose our data.
+* Testcontainers to run a Postgres instance for local dev, and tests.
+
+The application will expose a GraphQL endpoint at /graphql for the data to be consumed and modified.
+
+== What you will need
+
+To complete this guide, you will need the following:
+
+* Some time on your hands
+* A decent text editor or IDE
+* JDK 11 or greater installed with `JAVA_HOME` configured appropriately
```

Comment:
Why JDK 11? Either way you should use common-requirements.adoc or common-requirements-jdk11.adoc here

Reply:
I saw the generated build in `build/code/micronaut-graphql-todo/micronaut-graphql-todo-gradle-java/build.gradle` contained 

```
java {
    sourceCompatibility = JavaVersion.toVersion("11")
    targetCompatibility = JavaVersion.toVersion("11")
}
```

I will investigate, but I've removed this 11 requirement from the docs for now